vue-social/public/index.html

```diff
@@ -17,4 +17,9 @@
     </div>
     <!-- built files will be auto injected -->
   </body>
+
+  <script src="https://code.jquery.com/jquery-3.2.1.slim.min.js" integrity="sha384-KJ3o2DKtIkvYIK3UENzmM7KCkRr/rE9/Qpg6aAZGJwFDMVNA/GpGFF93hXpG5KkN" crossorigin="anonymous"></script>
```

Comment:
try to use npm library in package.json

Reply:
We are already using bootstrap-vue using npm. I will try to convert the navbar to use that so that we do not require this dependency

Reply:
Done